Bubble_Sort

```diff
@@ -0,0 +1,42 @@
+# Bubble Sort Algorithm ASC
+N = 3
+a = ["test", "My", "words"]
+print(a)
+
+for i in range(N-1):
+    for j in range(N-1-i):
+        if a[j] > a[j+1]:
+            a[j], a[j+1] = a[j+1], a[j]
+
+print(a)
+
+# Bubble Sort Algorithm DES
+def bubble(array):
+    iterations = len(array) - 1
+    for i in range(iterations):
+        for j in range(iterations-i):
+            if array[j] < array[j+1]:
+                array[j], array[j+1] = array[j+1], array[j]
+
+
+a = [5, 10, 99, 85, 12, 1, 78, 66, 4, 8]
+print(a)
+bubble(a)
+print(a)
+
+# Bubble Sort Algorithm With early stopping
+def bubble_sort(array):
```

Comment:
The function parameter (array) shadows variable name in line 40
It is better to use the different names